src/manifolds/SymmetricPositiveDefiniteFixedDeterminant.jl

```diff
@@ -0,0 +1,152 @@
+@doc raw"""
+    SymmetricPositiveDefiniteFixedDeterminant{N,D} <: AbstractDecoratorManifold{ℝ}
+
+The manifold of symmetric positive definite matrices of fixed determinant ``d > 0``, i.e.
+
+````math
+\mathcal P_d(n) =
+\bigl\{
+p ∈ ℝ^{n × n} \ \big|\ a^\mathrm{T}pa > 0 \text{ for all } a ∈ ℝ^{n}\backslash\{0\}
+  \text{ and } \det(p) = d
+\bigr\}.
+````
+This manifold is modelled as a submanifold of [`SymmetricPositiveDefinite`](@ref)`(n)`.
+
+The tangent space is modelled the same as for [`SymmetricPositiveDefinite`](@ref)`(n)`
+and consists of all symmetric matrices with zero trace
+```math
+    T_p\mathcal P_d(n) =
+    \bigl\{
+        X \in \mathbb R^{n×n} \big|\ X=X^\mathrm{T} \text{ and } \operatorname{tr}(p) = 0
+    \bigr\},
+```
+since for a constant determinant we require that `0 = D\det(p)[Z] = \det(p)\operatorname{tr}(p^{-1}Z)` for all tangent vectprs ``Z``.
```

Comment:
```suggestion
since for a constant determinant we require that `0 = D\det(p)[Z] = \det(p)\operatorname{tr}(p^{-1}Z)` for all tangent vectors ``Z``.
```